### PR TITLE
Add get_version package.

### PR DESCRIPTION
**Pull request by grst**
Checklist

- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L57-L66) for an example)
- [x] Source is from official source
- [x] Package does not vend other packages
- [x] Build number is 0
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your
      recipe (see
      [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos)
      for more details)
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there

### CODE REVIEW

**Comment by grst** [comment]
@flying-sheep, are you ok being listed as maintainer?

File: recipes/get_version/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+  version: {{ version }}
+
+source:
+  url:  https://github.com/flying-sheep/{{ name }}/archive/v{{ version }}.tar.gz
+  sha256: ed320df3b0604ecab925f5e00290c09f85c169bc8f826bb263b0d82889be487c
+
+build:
+  noarch: python
+  number: 0
+  script: "{{ PYTHON }} -m flit install --deps none"
```
Comment (marcelotrevisani) [inline recipes/get_version/meta.yaml]: Why ``flit``? should it be :
```suggestion
  script: "{{ PYTHON }} -m pip install . -vv
```

Comment (flying-sheep) [reply recipes/get_version/meta.yaml]: Why not flit? This is one more indirection and flit already does everything.

Comment (grst) [reply recipes/get_version/meta.yaml]: and using `pip` it fails on windows due to a `UnicodeDecodeError`. 
See [appveyor build](https://ci.appveyor.com/project/conda-forge/staged-recipes/builds/27555974)

Comment (grst) [reply recipes/get_version/meta.yaml]: and using `pip` it fails on windows due to a `UnicodeDecodeError`. 
See [appveyor build](https://ci.appveyor.com/project/conda-forge/staged-recipes/builds/27555974)

File: recipes/get_version/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+
+requirements:
+  host:
+    - python >=3.6
+    - pip
```
Comment (chrisburr) [inline recipes/get_version/meta.yaml]: Do you need `pip` if you're using `flit`?
```suggestion
```

Comment (grst) [reply recipes/get_version/meta.yaml]: `pip` is a dependency of `flit`, so no, not explicitly.

**Comment by grst** [comment]
Thanks, Chris!

**Pull request merged**